src/handlers/askingReplyRequestReason.js

```diff
@@ -8,29 +8,78 @@ export default async function askingArticleSubmission(params) {
   if (event.input !== 'n') {
     const reason = event.input;
 
+    const text =
+      `以下是您所填寫的理由：\n「\n${reason}\n」\n` +
+      '我們即將把您填寫的理由送至資料庫。' +
+      '若您送出的訊息或理由意味不明、造成闢謠編輯的困擾，可能會影響到您未來送出文章的權利。' +
+      '若要確認請輸入「y」、若不想填寫請輸入「n」、若要重新填寫理由請輸入「r」';
+
     replies = [
       {
-        type: 'text',
-        text: `以下是您所填寫的理由：\n「\n${reason}\n」`,
-      },
-      {
-        type: 'text',
-        text:
-          '我們即將把此訊息與您填寫的理由送至資料庫。若您送出的訊息或理由意味不明、' +
-          '造成闢謠編輯的困擾，可能會影響到您未來送出文章的權利。',
-      },
-      {
-        type: 'template',
-        altText:
-          '若要確認請輸入「y」、若要放棄請輸入「n」、若要重新填寫理由請輸入「r」',
-        template: {
-          type: 'buttons',
-          text: '請確認',
-          actions: [
-            createPostbackAction('明白我要送出', 'y', issuedAt),
-            createPostbackAction('重寫送出的理由', 'r', issuedAt),
-            createPostbackAction('我不想填理由', 'n', issuedAt),
-          ],
+        type: 'flex',
+        altText: text,
+        contents: {
+          type: 'bubble',
+          styles: {
+            footer: {
+              separator: true,
+            },
+          },
+          body: {
+            type: 'box',
+            layout: 'vertical',
+            contents: [
+              {
+                type: 'text',
+                text: '把您的理由送進資料庫',
+                weight: 'bold',
+                color: '#1DB446',
+                size: 'sm',
+              },
+              {
```

Comment:
Got it, I missed the case that `searchedText` can be different from the selected article.

I think it is ok to query the DB again for this purpose. Since we are using ID to get article instead of doing a DB search, API server's loading won't be affected a lot.

Btw instead of "訊息" shown in the image, I prefer it being more specific, such as `您之前選的訊息` or something similar.